tests/main/snapd-snap/task.yaml

```diff
@@ -154,3 +154,18 @@ execute: |
     unsquashfs -ll snapd_spread-test.snap | MATCH etc/ld.so.conf
     echo "Ensure we have libc"
     unsquashfs -ll snapd_spread-test.snap | MATCH libc.so
+
+    echo "Ensure we have apparmor_parser"
+    unsquashfs -ll snapd_spread-test.snap | MATCH usr/lib/snapd/apparmor_parser
+
+    echo "Ensure we can install the snapd snap"
+    snap set core experimental.snapd-snap=true
```

Comment:
this is not needed

Reply:
Thanks, I removed this in https://github.com/snapcore/snapd/pull/9939/commits/56a4c7a73c91b32137b41d9466a023c05f282084